README.md

```diff
@@ -15,6 +15,15 @@ KubeArmor leverages [Linux security modules \(LSMs\)](https://en.wikipedia.org/w
 
 ![KubeArmor High Level Design](.gitbook/assets/kubearmor_overview.png)
 
+## Use Cases
+### Kubernetes Runtime Security
+Kubearmor provides runtime protection for the Kubernetes workloads. It can restrict the runtime behavior of the pods/containers running on the k8s node at the system level using the Linux Security Modules. Apparmor, SElinux and BPF LSM are currently supported LSMs by Kubearmor. It Provides easy-to-use Policy semantics using Kubernetes CRDs. K8s custom resource `KubeArmorPolicy` can be used by the user to specify the security policy for the container/pod. 
+### Unorchestrated Container Runtime Security
+Kubearmor as a runtime security engine can also protect the non-orchestrated containerized workload running directly on a host/VM. Kubearmor can provide visibility as well as protection for process, file and network operations in the container workloads. It makes Kubearmor a perfect candidate for a runtime security solution to protect the IoT Edge containerized workloads. Kubearmor is well tested with the [Open Horizon](https://www.lfedge.org/projects/openhorizon/), an edge computing platform. This [post](https://github.com/kubearmor/KubeArmor/wiki/KubeArmor-to-protect-IoT-Edge-containerized-workloads) provides more information on Kubearmor to protect IoT edge container workloads on Open Horizon platform.
```

Comment:
> It makes Kubearmor a perfect candidate for a runtime security solution to protect the IoT Edge containerized workloads. Kubearmor is well tested with the [Open Horizon](https://www.lfedge.org/projects/openhorizon/), an edge computing platform. This [post](https://github.com/kubearmor/KubeArmor/wiki/KubeArmor-to-protect-IoT-Edge-containerized-workloads) provides more information on Kubearmor to protect IoT edge container workloads on Open Horizon platform.

Can this be rephrased? Because visibility and enforcement is not justifying KubeArmor's use in IOT platforms.